OrbitLinuxTracing/PerfEventRingBuffer.h

```diff
@@ -55,6 +61,8 @@ class PerfEventRingBuffer {
   void ReadAtTail(void* dest, uint64_t count) { return ReadAtOffsetFromTail(dest, 0, count); }
 
   void ReadAtOffsetFromTail(void* dest, uint64_t offset_from_tail, uint64_t count);
+
+  void ConsumeRecord(const perf_event_header& header, void* record, size_t size);
```

Comment:
Could you move it up before `ReadAtTail` (it keeps the current order and reflects the order in the .cpp file)?